petab/lint.py

```diff
@@ -14,6 +15,17 @@ def _check_df(df, req_cols, name):
         raise AssertionError(
             f"Dataframe {name} requires the columns {missing_cols}.")
 
+def assert_presence_of_empty_spaces(names_list, name):
```

Comment:
Please add a test case for this function

Reply:
I just added the test in `tests/test_lint.py`
could you have a look?